runtime/js/40_process.js

```diff
@@ -354,7 +354,12 @@ class ChildProcess {
     if (this.#waitComplete) {
       throw new TypeError("Child process has already terminated");
     }
-    op_spawn_kill(this.#rid, signo);
+    try {
```

Comment:
@yuxi-ovo I do not believe this fixes the issue. The error that was thrown in the issue is what is produced by line 355. Swallowing the error should be done on line 299 of this file :). The other option is to suppress the error entirely here but that is a discussion for the maintainers